src/components/common/Identifier/IdentifierInput.tsx

```diff
@@ -16,7 +16,7 @@ export interface IdentifierProps {
 const IdentifierInput = ({
   onIdentifierChange,
   canEdit,
-  namingLabel = "Value Set / Direct Reference Code",
+  namingLabel = "Naming System",
```

Comment:
Should this be changed to all instances of Naming System ?

Reply:
MAT-6284 only mentioned Identifier